src/nanosvg.h

```diff
@@ -2902,7 +2897,6 @@ NSVGimage* nsvgParseFromFile(const char* filename, const char* units, float dpi)
 error:
 	if (fp) fclose(fp);
 	if (data) free(data);
-	if (image) nsvgDelete(image);
```

Comment:
deleting this is technically correct, but i suspect you're overlooking the idiomatic intent.

Reply:
It correct. And indeed in this case it make sense. But it is not consistent across the fine to use `error` label to clean up. And keeping dead code doesn't make it easier to read. I don't mind either way and can revert this change if that's required.